webpack/webpack.common.config.js

```diff
@@ -16,8 +16,6 @@ fs.writeFileSync(configFile, JSON.stringify(config));
 // style files regexes
 const cssRegex = /\.css$/;
 const cssModuleRegex = /\.module\.css$/;
-const sassRegex = /\.(scss|sass)$/;
-const sassModuleRegex = /\.module\.(scss|sass)$/;
```

Comment:
🤔 why?

Reply:
Both are unused variables. I found then when I searched the project for sass and it turned out we have nothing to do with sass

Reply:
Ey! But we do have `scss` files, right?

Reply:
Are they processed by the style-loader and or css-loader here https://github.com/iov-one/wallet-demo/blob/v0.12.2/webpack/webpack.common.config.js#L80-L101?

Reply:
Correct!